web/turkey/survey/static/survey/js/auditors/within_typing_delay.js

```diff
@@ -0,0 +1,39 @@
+var AuditorWithinTypingDelay = {
+    start_date: new Date(),
+    first_typing_event_date: null,
+    // within_typing_delay: false,
+    typing_delay: 10000, // milliseconds
+    log_keydown_typing_event: function (e) {
+        // there has not yet been a typing event recorded
+        // and this typing event qualifies for recording
+        // because it is one of the letters of the alphabet
+        if (!this.first_typing_event_date
```

Comment:
Typing event criteria should match between auditors right? Either we're restricting to a key range or not.